osbs/build/user_params.py

```diff
@@ -312,14 +316,13 @@ class BuildUserParams(BuildCommon):
 
     KIND = USER_PARAMS_KIND_IMAGE_BUILDS
 
-    def __init__(self, build_json_dir=None, customize_conf=None):
+    def __init__(self, build_json_store=None):
         # defines image_tag, koji_target, filesystem_koji_task_id, platform, arrangement_version
-        super(BuildUserParams, self).__init__(build_json_dir=build_json_dir)
+        super(BuildUserParams, self).__init__(build_json_store=build_json_store)
+        self.repo_info = None
         self.base_image = BuildParam('base_image', allow_none=True)
         self.build_type = BuildParam('build_type')
         self.compose_ids = BuildParam("compose_ids", allow_none=True)
-        self.customize_conf_path = BuildParam("customize_conf", allow_none=True,
```

Comment:
Why this is not needed anymore?

Reply:
because this:
```
    # Override
    @property
    def customize_conf(self):
        raise RuntimeError('customize_conf not supported in BuildRequestV2')
```

we haven't support customize_conf in a while.

Reply:
went back and supported customize_conf because it was easier than rewriting dozens of unit test cases.

Reply:
Fixed.